lib/svg-sprite/config.js

```diff
@@ -324,10 +324,7 @@ module.exports = class SVGSpriterConfig {
         ['padding'].forEach(function(property) {
             let spacing;
 
-            if (!Array.isArray(this.spacing[property])) {
```

Comment:
Was this eslint's correction?

Reply:
Yeah, I guess the no negated condition rule I guess added some fixes

Reply:
Yeah, I guess the no negated condition rule I guess added some fixes